actor-client/src/test/scala/sbt/client/actors/FakeSbtClient.scala

```diff
@@ -170,6 +170,10 @@ final class FakeSbtClient(refFactory: ActorRefFactory,
 
   def lookupScopedKey(name: String): Future[Seq[protocol.ScopedKey]] = scopedKeyLookup(name)
 
+  def listSettings(): Future[Seq[protocol.ScopedKey]] = ???
```

Comment:
OK.  I'll add a task for me to go back and add some updates for the actor client.